src/server.ts

```diff
@@ -185,6 +187,15 @@ app.put('/clock', (req, res) => {
   res.sendStatus(200);
 });
 
+app.get('/scene/:sceneId.png', (req, res) => {
+  HueAPI.request('GET', `/scenes/${req.params.sceneId}`, {})
+    .then(scene => {
+      console.log(scene);
```

Comment:
Her vil jeg gerne have scene.lightStates men det kan jeg ikke.

Property 'lightStates' does not exist on type 'unknown'.